### PR TITLE
Update: Add search for raw query input during import

### PR DESCRIPTION
**Pull request by kevinhinterlong**
## Description
During the dimension bulk import, if a user pasted a dimension id containing a comma ('yes, comma'), then it will.
- paste 'yes, comma'
- bring up bulk import
- show 2 invalid values

## Proposed Changes
Add a search for the raw input in the bulk import
- paste 'yes, comma'
- bring up bulk import
- show 2 invalid values _and_ button allowing import of raw value

## Screenshots
bulk-import-raw-value-search

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by mattgraham1995** [comment]
A `raw input` field showing what exactly was pasted in would be helpful

**Comment by kevinhinterlong** [comment]
Pasted input preview

File: packages/reports/addon/templates/components/power-select-bulk-import-trigger.hbs [hunk 1]
```diff
@@ -44,13 +44,12 @@
...
-    onSelectValues=(pipe (action "importValues") (action (set this "_showBulkImport" false)))
-    onCancel=(action (set this "_showBulkImport" false))
-  }}
-{{/navi-modal}}
+<NaviModal @isShown={{this._showBulkImport}}>
+  <DimensionBulkImport
+    @dimension={{@extra.filter.subject}}
+    @queryIds={{this._bulkImportQueryIds}}
+    @rawQuery={{this._bulkImportRawValue}}
+    @onSelectValues={{pipe (action "importValues") (set this._showBulkImport false)}}
```
Comment (mattgraham1995) [inline packages/reports/addon/templates/components/power-select-bulk-import-trigger.hbs]:
```suggestion
    @onSelectValues={{queue (action "importValues") (set this._showBulkImport false)}}
```

File: packages/reports/tests/integration/components/dimension-bulk-import-test.js [hunk 2]
```diff
@@ -317,4 +315,25 @@ module('Integration | Component | Dimension Bulk Import', function(hooks) {
...
+    this.setProperties({
+      dimension: { name: 'commaDim', longName: 'Dimension With Comma' },
+      onSelectValues: () => {},
+      onCancel: () => {},
+      rawQuery,
+      queryIds: rawQuery.split(',').map(s => s.trim())
+    });
+
+    await render(COMMON_TEMPLATE);
+
```
Comment (mattgraham1995) [inline packages/reports/tests/integration/components/dimension-bulk-import-test.js]: Can we assert that the rawQuery is shown?

**Pull request merged**